tests/acceptance/30_custom_promise_types/12_multiple_promise_types_same_module.cf

```diff
@@ -0,0 +1,142 @@
+######################################################
+#
+#  Basic test of custom promise types / promise modules with a custom promise
+#  module used for two promise types.
+#
+#####################################################
+body common control
+{
+    inputs => { "../default.cf.sub" };
+    bundlesequence  => { default("$(this.promise_filename)") };
+    version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  files:
+    "$(G.testfile)"
+      delete => init_delete;
+    "$(G.testfile)2"
+      delete => init_delete;
+}
+
+body delete init_delete
+{
+      dirlinks => "delete";
+      rmdirs   => "true";
+}
+
+#######################################################
+
+promise agent example
+{
+    interpreter => "/bin/bash";
+    path => "$(this.promise_dirname)/example_module.sh";
+}
+
+promise agent example_two
+{
+    interpreter => "/bin/bash";
+    path => "$(this.promise_dirname)/example_module.sh";
```

Comment:
Idea: We could make the example module save files with filenames based on promise type, promiser, etc. and then check that those files exist in the test.

Reply:
Added https://tracker.mender.io/browse/CFE-3657 for this.